src/main/java/marquez/api/JobRunState.java

```diff
@@ -1,168 +1,17 @@
 package marquez.api;
```

Comment:
This file isn't needed. We don't need to provide info about JobRunState in the API layer b/c we don't expose this info (the state can be retrieved through the JobRun object).

Reply:
Done.